src/mobile/src/ui/components/DeepLinking.js

```diff
@@ -0,0 +1,84 @@
+import React, { Component } from 'react';
+import { Linking } from 'react-native';
+import { withNamespaces } from 'react-i18next';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { initiateDeepLinkRequest, setSetting } from 'shared-modules/actions/wallet';
+import { parseAddress } from 'shared-modules/libs/iota/utils';
+import { generateAlert } from 'shared-modules/actions/alerts';
+import { changeHomeScreenRoute } from 'shared-modules/actions/home';
+
+export default () => (C) => {
+    class WithDeepLinking extends Component {
+        constructor() {
+            super();
+            this.setDeepUrl = this.setDeepUrl.bind(this);
+        }
+        componentWillMount() {
```

Comment:
Tiny nitpick:
```suggestion
        componentDidMount() {
```
`componentWillMount` will be deprecated in a future release of React, so the [docs](https://reactjs.org/docs/react-component.html#unsafe_componentwillmount) advise using `componentDidMount` for subscriptions